lib/wibox/widget/textclock.lua

```diff
@@ -24,16 +26,18 @@ end
 --
 -- @tparam[opt=" %a %b %d, %H:%M "] string format The time format.
 -- @tparam[opt=60] number timeout How often update the time (in seconds).
+-- @tparam[opt] string timezone The timezone to display, or localtime if nil.
```

Comment:
Fine by me. I was not sure how to document the default value since it's not like "we use the string `foo` if none is given".